src/reactapp/scripts/configurePaymentMethods.js

```diff
@@ -23,9 +23,18 @@ module.exports = (async () => {
     'payment method'
   );
 
-  let content = 'export default {};\n';
   if (paymentMethodList.length) {
-    content = paymentMethodList
+    let content = `/**
+ * 📢 📢 📢 This file is unlikely to be changed. Handle with care. 🙅🏽 🙅🏽 🙅🏽
+ *
+ * When you have custom payment repo integration, then the content of this file
+ * will be altered during performing "npm install". So, in most of the cases, you
```

Comment:
will be overwritten during performing "npm install". So, in most of the cases, you

Reply:
Applied the changes suggested.